Update README.md (#4)
nuxeo/nuxeo-studio-community-cookbook: modules/nuxeo/modeler-tips-tricks/README.md

`ecm:mixinType != 'HiddenInNavigation' AND ecm:isCheckedInVersion = 0 AND ecm:currentLifeCycleState != 'deleted' AND ecm:fulltext LIKE '?'`

And uncheck "quote parameter"

Ignore the validation warning "Number of parameters and number of occurrences of the '?' character in the query filter do not match"

> The, you have to create a XML extension:

Reviewer: ```suggestion
Then, you have to create an XML extension:
```
